hadoop-hdds/container-service/src/test/java/org/apache/hadoop/ozone/container/keyvalue/TestKeyValueContainerCheck.java

```diff
@@ -120,10 +132,70 @@ public TestKeyValueContainerCheck(String metadataImpl) {
     container.close();
 
     // next run checks on a Closed Container
-    valid = kvCheck.fullCheck();
+    valid = kvCheck.fullCheck(new DataTransferThrottler(
+        HddsConfigKeys.HDDS_CONTAINER_SCANNER_VOLUME_BYTES_PER_SECOND_DEFAULT),
+        null);
     assertTrue(valid);
   }
 
+  /**
+   * Sanity test, when there are corruptions induced.
+   * @throws Exception
+   */
+  @Test
+  public void testKeyValueContainerCheckCorruption() throws Exception {
```

Comment:
you can run this test under memory profiler to make sure there are no leaks. Thanks

Reply:
/label ozone

Reply:
If you add a label called ozone, then this pull request will be processed by Jenkins. Just FYI. I have done that for this patch.

Reply:
sure will do.